stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/mentions/MentionsListViewModel.kt

```diff
@@ -25,7 +25,7 @@ public class MentionsListViewModel : ViewModel() {
     )
 
     private companion object {
-        const val QUERY_LIMIT = 30
+        private const val QUERY_LIMIT = 30
```

Comment:
Turns out `const` values inside companion objects are exposed as a public field in the bytecode, so they're worth making doubly `private`...

Reply:
To clarify, this is a bug that's being tracked here https://youtrack.jetbrains.com/issue/KT-25009